sql/item_equipment.sql

```diff
@@ -14159,17 +14159,17 @@ INSERT INTO `item_equipment` VALUES (28644,'ogapepo_cape',99,0,1753628,0,0,0,327
 INSERT INTO `item_equipment` VALUES (28645,'ogapepo_cape_+1',99,0,1753628,0,0,0,32768,0);
 INSERT INTO `item_equipment` VALUES (28646,'radical_mantle',99,0,2473971,0,0,0,32768,0);
 INSERT INTO `item_equipment` VALUES (28647,'radical_mantle_+1',99,0,2473971,0,0,0,32768,0);
-INSERT INTO `item_equipment` VALUES (28648,'priwen',99,119,64,0,3,0,2,0);
+INSERT INTO `item_equipment` VALUES (28648,'priwen',99,119,64,58,3,0,2,0);
 INSERT INTO `item_equipment` VALUES (28649,'rinda_shield',99,119,193,30,3,0,2,0);
-INSERT INTO `item_equipment` VALUES (28650,'she-slime_shield',1,0,4194303,0,1,0,2,0);
-INSERT INTO `item_equipment` VALUES (28651,'metal_slime_shield',1,0,4194303,648,1,0,2,0);
+INSERT INTO `item_equipment` VALUES (28650,'she-slime_shield',1,0,4194303,649,1,0,2,0);
+INSERT INTO `item_equipment` VALUES (28651,'metal_slime_shield',1,0,4194303,650,1,0,2,0);
 INSERT INTO `item_equipment` VALUES (28652,'hatchling_shield',1,0,4194303,651,1,0,2,0);
 INSERT INTO `item_equipment` VALUES (28653,'mundus_shield',1,0,4194303,644,1,0,2,0);
 INSERT INTO `item_equipment` VALUES (28654,'beatific_shield_+1',99,119,2385,646,1,0,2,0);
-INSERT INTO `item_equipment` VALUES (28655,'slime_shield',1,0,4194303,0,1,0,2,0);
+INSERT INTO `item_equipment` VALUES (28655,'slime_shield',1,0,4194303,648,1,0,2,0);
 INSERT INTO `item_equipment` VALUES (28656,'eminent_shield',99,117,193,25,3,0,2,0);
 INSERT INTO `item_equipment` VALUES (28657,'kaidate',99,109,2529,32,4,0,2,0);
-INSERT INTO `item_equipment` VALUES (28658,'sors_shield',99,109,3719388,0,2,0,2,0);
+INSERT INTO `item_equipment` VALUES (28658,'sors_shield',99,109,3719388,36,2,0,2,0);
```

Comment:
just spotchecked this out - it's showing as a standard round shield vs the blue sors shield. I still think having a model id is better than not but this one is not fully correct.